contentcuration/contentcuration/frontend/shared/views/FullscreenModal.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+
+  <VDialog
+    ref="dialog"
+    :value="value"
+    app
+    attach="body"
+    fullscreen
+    scrollable
+    persistent
+    transition="dialog-bottom-transition"
+    v-bind="$attrs"
+  >
+    <VCard style="overflow-y: auto;">
+      <VToolbar
+        :color="color"
+        dark
+        fixed
+        :extension-height="48"
+        clipped-left
+        clipped-right
+      >
+        <VToolbarItems>
+          <slot name="close">
+            <VBtn flat icon exact data-test="close" @click="$emit('input', false)">
+              <Icon>clear</Icon>
+            </VBtn>
+          </slot>
+        </VToolbarItems>
+        <VToolbarTitle>
+          <slot name="header">
+            {{ header }}
+          </slot>
+        </VToolbarTitle>
+        <VSpacer />
+        <slot name="action"></slot>
+        <template v-if="$slots.tabs" #extension>
+          <VTabs
+            color="primary"
+            slider-color="white"
+            align-with-title
+          >
+            <slot name="tabs"></slot>
+          </VTabs>
+        </template>
+      </VToolbar>
+      <OfflineText toolbar :offset="topToolbarHeight" />
```

Comment:
So I tried to use their app attribute, but the problem there is that it affects the layout of the main page because it's used on a modal (everything outside of the modal also shifts to the sizing on the modal). This is sort of a way to get the same behavior without causing problems (also some weird handling because there are multiple toolbars on the page

Reply:
Yeah, I think this may be where my decision to use nested routes is not helping us at all. I think we had tentatively decided to go back on this previously, but don't think we've actually acted on it.

If each full screen modal was its own independent route, then it wouldn't have the same issue of the `app` attribute causing issues on the page it is covering.

Reply:
That would be nice! Next pass :)